qadence/backend.py

```diff
@@ -114,6 +114,7 @@ class Backend(ABC):
     is_remote: bool
     with_measurements: bool
     native_endianness: Endianness
+    engine: Engine
```

Comment:
can we add an entry in the docstring for this one? weird that the docs linting is not failing here, actually.

Reply:
oh good catch, yeah, indeed weird..

Reply:
actually several other, non-MR related args are missing too